hasjob/forms/filterset.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+
+from wtforms.ext.sqlalchemy.fields import QuerySelectMultipleField
+from wtforms.widgets import CheckboxInput, ListWidget
+from baseframe import __
+import baseframe.forms as forms
+from ..models import CURRENCY, JobType, JobCategory
+
+
+def format_geonameids(geonameids):
+    if not geonameids:
+        return []
+    return [int(geonameid) for geonameid in geonameids]
+
+
+def get_currency_choices():
+    choices = [('', __('None'))]
+    choices.extend(CURRENCY.items())
+    return choices
+
+
+class FiltersetAssocationsForm(forms.Form):
+    types = QuerySelectMultipleField(__("Job types"),
+        widget=ListWidget(), option_widget=CheckboxInput(),
+        query_factory=lambda: JobType.query.order_by('title'), get_label='title',
```

Comment:
This will need to be joined with `BoardJobType`. No point offering job types not available in the current board.